packages/pn-pa-webapp/src/App.tsx

```diff
@@ -255,6 +257,17 @@ const ActualApp = () => {
         onAssistanceClick={handleAssistanceClick}
         isLogged={!!sessionToken}
       >
+        <DisclaimerModal
```

Comment:
This modal was created for the contacts section and it will be removed with the rework. Use PnDialog directly

Reply:
insert pnDialog